src/app/utils/field-validation/index.ts

```diff
@@ -70,7 +74,13 @@ const isResponsePresentOnHiddenField = (
       return true
     }
   } else if (isProcessedAttachmentResponse(response)) {
-    if (response.filename.trim() !== '') {
+    if (
+      (response.filename &&
+        typeof response.filename === 'string' &&
+        response.filename.trim() !== '') ||
+      response.answer.trim() !== '' ||
+      response.content
```

Comment:
why might the type of `filename` not be `string`? if we're worried that it might be undefined, you can use `response.filename?.trim() !== ''`. you don't need `response.filename && typeof response.filename === 'string'` because if `filename` is `undefined` then its type will also be `undefined`, and if it's an empty string then there's no harm calling `trim()` on it. and the `response.answer.trim()` is repeated.

could you also add some explanatory comments about what's going on here? I'm not sure what the behaviour is meant to be if `filename` is undefined.

Reply:
Added comment that filename is undefined when there is no response. I also removed the string check, but I think we still need `response.filename && response.filename.trim()` because if filename is undefined, then `response.filename.trim() !== ''` returns true, which is not what we want

Reply:
sorry I don't quite understand what you're saying here

> because if filename is undefined, then response.filename.trim() !== '' returns true, which is not what we want

if filename is undefined, `response.filename.trim()` will throw a null pointer exception, not return true. again, if your goal is to check that filename is defined before calling `trim` on it, you can do `response.filename?.trim() !== ''` - note the `?` for optional chaining.

Reply:
Sorry, I meant `response.filename?.trim() !== '' returns true`

Reply:
okie as discussed on slack - 

> return true only if response.filename is defined and response.filename.trim() !== ''